build-tools/packages/build-cli/src/handlers/promptFunctions.ts

```diff
@@ -335,6 +335,7 @@ export const promptToReleaseDeps: StateHandlerFunction = async (
             prompt.sections.push({
                 title: "FIRST",
                 message: `Release these packages first:\n\n${chalk.blue(packageSection)}`,
+                cmd: `flub bump deps ${packageSection} --updateType latest`,
```

Comment:
The instructions are to release packages, so the commands should be `flub release -g/-p <PACKAGE>`. Maybe instead of having a single step for packages and one for release groups, maybe put each package/release group in its own section. I think that will be clearest to the user.